apm-agent-plugins/apm-apache-httpclient/apm-apache-httpclient5-plugin/src/test/java/co/elastic/apm/agent/httpclient/v5/ApacheHttpAsyncClientInstrumentationTest.java

```diff
@@ -181,17 +168,16 @@ public void testSpanFinishWithIllegalUrl() throws Exception {
     }
 
     /**
-     * Difference between sync and async requests is that
-     * In async requests you need {@link SimpleRequestBuilder#setAuthority(URIAuthority)} explicitly
-     * And in this case exception will be thrown from {@link org.apache.hc.client5.http.impl.async.AsyncProtocolExec#execute}
+     * RFC 7230: treat presence of userinfo in authority component in request URI as an HTTP protocol violation.
      * <p>
-     * SimpleHttpRequest req = SimpleRequestBuilder.get().setPath(path)
-     * .setScheme("http")
-     * .setAuthority(new URIAuthority(uri.getUserInfo(), uri.getHost(), uri.getPort()))
-     * .build();
+     * Uses {@link org.apache.hc.core5.http.message.BasicHttpRequest#setUri} to fill {@link org.apache.hc.core5.net.URIAuthority}
+     * <p>
+     * Assertions on authority in {@link org.apache.hc.client5.http.impl.classic.ProtocolExec#execute}
      */
     @Override
     public boolean isTestHttpCallWithUserInfoEnabled() {
-        return true;
+        // earlier 5.x versions still allowed user info for async requests, however this has been removed
+        // thus it's not relevant to keep testing for it.
+        return false;
```

Comment:
[for reviewer] this is no longer supported with recent versions like 5.4.4, hence it's not really worth keeping tests for it.